### ISSUE TITLE
Message "An account already exists..." incorrectly displayed

### ISSUE THREAD

**Issue by lem9** [phpmyadmin/phpmyadmin#11720]
In User accounts > Edit privileges (for one account) > Login Information, this message is displayed on entering the panel:
````
An account already exists with the same username but possibly a different hostname.
````
The goal is to warn about a conflict when creating an account, but here the user has just selected an account to possibly change login information.

**Comment by devenbansod** [comment]
I am not sure, but is it not the 'Change password' tab to change the login information ?

One can change the 'password', 'authentication plugin/password hashing' on the 'Change password' tab, 
and only if you want to change the 'host' for a user, then that warning is suitable I guess.

**Comment by ibennetch** [comment]
Possibly somewhat related to #11607

**Comment by lem9** [comment]
@devenbansod The scenario I described brings the "Change login information / Copy user account" panel which can be used for many things, for example a hostname change. It can also be used to change password stuff for the same user or for the copied user.

**Comment by devenbansod** [comment]
@lem9 : yes. I guess I understand now. As Madhura fixed #11607, it is now clear that if you want to *only edit the password*, you would have to use the 'Change Password' tab (panel) **only**.

The panel you mentioned i.e. "Change login information / Copy user account" can be used for :
1. Changing hostname for the same user.
2. Copying this user to a different user, possibly with different username, hostname, password, auth_plugin etc. 

In the second case, I think that if someone just wants to change the hostname for the current user or may be copy this user with only hostname changed, then this warning is suitable (but I am not very sure).

Also then, IMO we should be check for an existing user and show the warning, but *only when you have edited the username field** and **now the new value clashes with an existing user*. (this would be the behavior for both the above situations)

**Comment by lem9** [comment]
@devenbansod I agree with your suggestion about editing and clashing. So *on entering this panel*, the warning does not make sense.
